web/uk-base-world/scripts/start.sh

```diff
@@ -98,30 +97,35 @@ then
 
     # Copy in the stack uploader config(s)
     cp "./inputs/config/dukes_2023.json" "$UPLOAD_CONFIG/"
+    cp -r ./inputs/config/pylons-and-veg/* "$UPLOAD_CONFIG/"
+
+    # Run the stack manager to start a new stack
+    cd "$ROOT/Deploy/stacks/dynamic/stack-manager" || exit
+    echo "Running the stack start up script..."
+    ./stack.sh start UKBASEWORLD 38383
 
     # Copy in the data for upload
     UPLOAD_DATA="$ROOT/Deploy/stacks/dynamic/stack-data-uploader/inputs/data"
+    echo "Copying data..." 
     cp -r "./inputs/data/." "$UPLOAD_DATA/" 
```

Comment:
The `cd` command here affects the `cp` call later, causing it to copy incorrect directories into the stack uploader. Either need to cd back to $START before line 110, or update the source directory of the `cp` call to `"$START/inputs/data/."`.